projects/astriddao/index.js

```diff
@@ -0,0 +1,64 @@
+const sdk = require("@defillama/sdk");
+// In case BigInt is not available.
+const BN = require("bn.js");
+
+const ActivePoolABI = require("./abi/ActivePool.abi.json");
+const DefaultPoolABI = require("./abi/ActivePool.abi.json");
+
+const DECIMAL_PRECISION = new BN("1000000000000000000");
+
+function getBNFromCallResponse(response) {
+  if ("output" in response) {
+    return new BN(response.output);
+  }
+  return new BN(0);
+}
+
+const CONTRACT_ADDRESSES = {
+  // Pools holding ASTR.
+  ACTIVE_POOL: "0x70724b57618548eE97623146F76206033E67086e",
```

Comment:
The only token I see in this pool is `0x19574c3c8fafc875051b665ec131b7e60773d2c9` and that is not the usual wrapped astar token address